src/common/ipc.c

```diff
@@ -72,65 +72,25 @@ read_strdata(int fd, size_t n, ...)
 	return 0;
 }
 
-static int
-write_strdata(int fd, size_t n, ...)
-{
-	uint16_t magic = IPC_MAGIC;
-	const char *cur;
-	size_t slen;
-	va_list ap;
-
-#define EMIT_BYTES(b, nb)          \
-	do {                           \
-		if (write(fd, b, nb) < nb) \
-			return -1;             \
-	} while (0)
-
-#define EMIT_VAR(v) EMIT_BYTES(&v, sizeof(v))
-
-	va_start(ap, n);
-	while (n--) {
-		cur = va_arg(ap, const char *);
-		slen = strlen(cur);
-
-		EMIT_VAR(slen);
-		EMIT_BYTES(cur, slen);
-		EMIT_VAR(magic);
-	}
-	va_end(ap);
-
-#undef EMIT_VAR
-#undef EMIT_BYTES
-
-	return 0;
-}
-
 int
 sosc_ipc_msg_write(int fd, sosc_ipc_msg_t *msg)
 {
+	uint8_t buf[64];
```

Comment:
i'd be more comfortable with bumping this up to 256

Reply:
64 is used throughout the code for similar cases, do you want to increase it just here or everywhere? how about a named constant in that case?

Reply:
good point actually. nevermind!